web/src/styles/base/global.scss

```diff
@@ -232,22 +232,23 @@ ol {
 
 .usa-link:visited,
 .usa-prose > a:visited {
-  color: $primary !important;
+  color: $accent-cool-darkest !important;
 }
 
 .usa-link:focus,
-.usa-prose > a:focus {
-  outline: 0.25rem solid $primary !important;
+.usa-prose > a:focus,
+.usa-prose a:focus {
+  outline: 0.25rem solid $accent-cool-darkest !important;
 }
 
 .usa-link:hover,
 .usa-prose > a:hover {
-  color: $primary-dark !important;
+  color: $accent-cool-more-dark !important;
 }
 
 .usa-link:active,
 .usa-prose > a:active {
-  color: $primary-darker !important;
+  color: $accent-cool-darkest !important;
```

Comment:
[pebble] We've only addressed anchor elements here. You'll need to update the unstyled button as well. There are instances in the app where Product desired analytics to fire when a link is clicked, to enable this, we used the unstyled button.

Reply:
The ticket says hyperlinks and opportunity cards. Unless I'm misremembering I don't think we discussed this applying to buttons.

Reply:
Gotcha. My misunderstanding. Included this.